roles/etcd/tasks/gen_certs.yml

```diff
@@ -40,7 +40,7 @@
                     {{ m }}
                   {% endif %}
                 {% endfor %}"
-    - HOSTS: "{% for h in (groups['k8s-cluster'] + groups['calico-rr']|default([]))|unique %}
+    - HOSTS: "{% for h in groups['k8s-cluster'] %}
```

Comment:
this discards @adidenko 's fixes for calico-rr (out of scope of CI, requires manual testing and confirm from @adidenko )